cpp/ps0003/main (3).cpp

```diff
@@ -1,16 +1,22 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
+    // prompting user for number 2/2
+    int num;
+    cout << "enter your num \n";
+    cin >> x;
```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number